conmon/conmon.c

```diff
@@ -1283,6 +1285,12 @@ int main(int argc, char *argv[])
 			NULL);
 	}
 
+	if (!opt_exec && opt_replace_listen_pid) {
+		add_argv(runtime_argv,
+			"--replace-listen-pid",
```

Comment:
Is `--replace-listen-pid` a `runc create` option?  If so, can't we drop the next hunk?

Reply:
No, it is not.

Reply:
> No, it is not.

Should it be?  If not, I think we should drop this `runtime_argv` append hunk.

Reply:
Thanks, I screwed up.  I was confused by your question, but now understand and removed the block.